metaworld/policies/sawyer_button_press_topdown_wall_v1_policy.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+
+from metaworld.policies.action import Action
+from metaworld.policies.policy import Policy, assert_fully_parsed, move
+
+
+class SawyerButtonPressTopdownWallV1Policy(Policy):
+
+    @staticmethod
+    @assert_fully_parsed
+    def _parse_obs(obs):
+        return {
+            'hand_pos': obs[:3],
+            'button_pos': obs[3:6],
+            'extra_info': obs[6:],
```

Comment:
if it's truly extra can we remove it?

Reply:
Avnish and KR wanted the length of the observation space to be consistent across environments, so I don't want to remove information from there. Here in `_parse_obs`, I could remove it, but right now there's code that asserts that `_parse_obs` covers everything returned by the observation: `assert(sum([piece_of_obs.length for ... in ...]) == obs.length)`.

I actually think it's useful to label it as `extra_info`. It shows that the information was intentionally unused and unnecessary for solving the env (as opposed to the author forgetting about it or something)

Reply:
Perhaps we could call it obj2 position_not_used or something like that to indicate that it's not used.